test/e2e/cluster.sh

```diff
@@ -102,17 +102,17 @@ if [ -n "${GINKGO_SKIP}" ]; then
     SKIP_AFTER_SCALE_UP="${GINKGO_SKIP}"
   fi
   if [ "${SCALE_CLUSTER}" = "true" ]; then
-    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}|${SKIP_AFTER_SCALE_DOWN}"
+    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}|${SKIP_AFTER_SCALE_DOWN}|${GINKGO_SKIP_AFTER_UPGRADE}"
```

Comment:
I think we need to check if `GINKGO_SKIP_AFTER_UPGRADE` is set and not empty before we use it here

Reply:
Updated